src/v0/destinations/criteo_audience/transform.js

```diff
@@ -0,0 +1,97 @@
+const {
+  BASE_ENDPOINT,
+  operation
+} = require("./config");
+const {
+  defaultRequestConfig,
+  removeUndefinedAndNullValues,
+  simpleProcessRouterDest,
+  defaultPatchRequestConfig
+} = require("../../util");
+const { InstrumentationError } = require("../../util/errorTypes");
+
+const { getAccessToken, preparePayload } = require("./util");
+
+const prepareResponse = (payload, audienceId, accessToken) => {
+  const response = defaultRequestConfig();
+  response.endpoint = `${BASE_ENDPOINT}audiences/${audienceId}/contactlist`;
+  response.body.JSON = removeUndefinedAndNullValues(payload);
+  response.method = defaultPatchRequestConfig.requestMethod;
+  response.headers = {
+    Authorization: `Bearer ${accessToken}`,
+    "Content-Type": "application/json",
+    Accept: "application/json"
+  };
+  return response;
+};
+/**
+ * This function is used for building the final response to be returned.
+ * @param {*} message
+ * @param {*} destination
+ * @returns
+ */
+const responseBuilder = async (message, destination, accessToken) => {
```

Comment:
Can you let me know if we're making any external api calls inside this method ? I don't see that happening. Let me know if there is a case where we are doing it

Reply:
We are not making axios calls during transformation. there is no case where we are doing it

Reply:
Can you remove `async` then ?

Reply:
Removing